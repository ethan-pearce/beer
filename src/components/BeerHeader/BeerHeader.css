.beerName{
    font-size: 24px;
    margin: 0;
}

.beerSubtitle{
    font-size: 16px;
    margin: 0 0 16px;
}

.organic {
    height: 40px;
    width: 40px;
}