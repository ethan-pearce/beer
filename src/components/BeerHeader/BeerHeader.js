import './BeerHeader.css'

const BeerHeader = ({title, subTitle, organic}) => {
return (
        <>
            <h2 className={'beerName'}>{title}</h2>
            <h3 className={'beerSubtitle'}>{subTitle}</h3>
            {organic && <img className={'organic'} alt={'organic'} src={'https://modernbuyerbehaviour.files.wordpress.com/2016/03/organic.jpg'}/>}
        </>
    );
}

export default BeerHeader;