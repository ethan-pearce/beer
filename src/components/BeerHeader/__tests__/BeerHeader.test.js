import React from 'react'
import { render } from '@testing-library/react';
import BeerHeader from '../BeerHeader'

describe('BeerHeader Component', () => {
    it('should render', () => {
        const { container } = render(<BeerHeader title={'test header'} subTitle={'test sub-header'} />)
        expect(container).toMatchSnapshot()
    })

    it('should render with Organic Icon', () => {
        const { container } = render(<BeerHeader title={'test header'} subTitle={'test sub-header'} organic={true} />)

        expect(container.getElementsByClassName('organic').length).toBe(1)
        expect(container).toMatchSnapshot()
    })
})