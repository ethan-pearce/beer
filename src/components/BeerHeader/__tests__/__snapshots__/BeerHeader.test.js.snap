// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BeerHeader Component should render 1`] = `
<div>
  <h2
    class="beerName"
  >
    test header
  </h2>
  <h3
    class="beerSubtitle"
  >
    test sub-header
  </h3>
</div>
`;

exports[`BeerHeader Component should render with Organic Icon 1`] = `
<div>
  <h2
    class="beerName"
  >
    test header
  </h2>
  <h3
    class="beerSubtitle"
  >
    test sub-header
  </h3>
  <img
    alt="organic"
    class="organic"
    src="https://modernbuyerbehaviour.files.wordpress.com/2016/03/organic.jpg"
  />
</div>
`;
